### PR TITLE
[Diffusion] Only import sgl_kernel in custom op cuda path (SiluAndMul and RMSNorm)

### PR DESCRIPTION
**Pull request by yeahdongcn**
## Motivation

This is also found in adapting `multimodal_gen` for MThreads GPUs. Avoid forcing optional or backend-specific kernels to be loaded eagerly.

## Modifications

Only import `sgl_kernel` in cuda code path.

## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.io/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.io/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.io/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.io/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.io/developer_guide/contribution_guide.html#benchmark-the-speed).
- [ ] Follow the SGLang code style [guidance](https://docs.sglang.io/developer_guide/contribution_guide.html#code-style-guidance).
- [ ] Work with maintainers to merge your PR. See the [PR Merge Process](https://github.com/sgl-project/sglang/blob/main/.github/MAINTAINER.md#pull-request-merge-process)

### CODE REVIEW

**Comment by yeahdongcn** [comment]
@mickqian Could you also review this one? Thanks.

**Comment by mickqian** [comment]
would you take a look? @BBuf

File: python/sglang/multimodal_gen/runtime/layers/activation.py [hunk 1]
```diff
@@ -9,8 +9,11 @@
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
-from sgl_kernel import silu_and_mul
 
+from sglang.multimodal_gen.runtime.platforms import current_platform
+
+if current_platform.is_cuda() or current_platform.is_hip():
```
Comment (BBuf) [inline python/sglang/multimodal_gen/runtime/layers/activation.py]:
```suggestion
_is_cuda = current_platform.is_cuda()
_is_hip = current_platform.is_hip()
if _is_cuda or _is_hip:
   from sgl_kernel import silu_and_mul
```

File: python/sglang/multimodal_gen/runtime/layers/layernorm.py [hunk 2]
```diff
@@ -8,7 +8,11 @@
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
-from sgl_kernel import fused_add_rmsnorm, rmsnorm
+
+from sglang.multimodal_gen.runtime.platforms import current_platform
+
+if current_platform.is_cuda():
```
Comment (BBuf) [inline python/sglang/multimodal_gen/runtime/layers/layernorm.py]: Same as above

**Review by BBuf — APPROVED** [review]
